cpa/parsing.go

```diff
@@ -139,3 +141,36 @@ func (err MultiError) Error() string {
 		return fmt.Sprintf("%d error(s) occurred: %s", len(err), strings.Join(messages, "; "))
 	}
 }
+
+//LoadPolicyFile takes policy file path as an input, and returns parsed policy
+func LoadPolicyFile(filePath string) (*Policy, error) {
+	documentBundle := map[string]string{}
+	fileContent, err := ioutil.ReadFile(filepath.Clean(filePath))
```

Comment:
I think filepath.Clean is redundant as ReadFile will sanitize its input for you. But you can check if this is true if you want. 95% sure

Reply:
is required